tasks/gaia.js

```diff
@@ -166,14 +166,15 @@ async function sendTokens(
   toAddress,
   chainId
 ) {
+  await sleep(500)
```

Comment:
Just wondering: why was this necessary before sending the money?

Reply:
I think this was due to the parallel signing issues. I will test this again. Nice find.